tests/FSharp.Compiler.ComponentTests/EmittedIL/Misc/AnonRecd.fs.il.net472.bsl

```diff
@@ -79,13 +79,13 @@
 
 } 
 
-.class public auto ansi serializable sealed beforefieldinit '<>f__AnonymousType1912756633`2'<'<A>j__TPar','<B>j__TPar'>
+.class public auto ansi serializable sealed beforefieldinit '<>f__AnonymousType3376895005`2'<'<A>j__TPar','<B>j__TPar'>
```

Comment:
What does this change do to the "By Default Works Across Assembly Boundaries " principle https://github.com/fsharp/fslang-design/blob/main/FSharp-4.6/FS-1030-anonymous-records.md#design-principle-by-default-works-across-assembly-boundaries of F# anonymous records?

So either:
- New code with new compiler working with a .dll that was built with the older version
- Or, older compiler working with .dll that was produced with the new one

(let's say the .dll exposes a function which expects a particular anonymous record)

Reply:
Yup, it does not work cross-assembly :(